Makefile

```diff
@@ -132,37 +132,25 @@ endif
 ##############################################################################
 
 .PHONY: docs
-docs: $(docs_data_files)
+docs:
 	@rm -rf build/html_docs
 	sh script/build_apm_docs.sh apm-server docs/index.asciidoc build
 
 .PHONY: update-beats-docs
 update-beats-docs: $(PYTHON)
 	@$(PYTHON) script/copy-docs.py
 
-docs_data_files=\
-  docs/data/intake-api/generated/events.ndjson \
-  docs/data/intake-api/generated/rum_v3_events.ndjson \
-  docs/data/intake-api/generated/sourcemap/bundle.js.map \
-  docs/data/elasticsearch/generated/errors.json \
-  docs/data/elasticsearch/generated/spans.json \
-  docs/data/elasticsearch/generated/transactions.json \
-  docs/data/elasticsearch/generated/metricsets.json \
-  docs/data/elasticsearch/generated/rum_v3_transactions.json \
-  docs/data/elasticsearch/generated/rum_v3_errors.json
-
-docs_data: $(docs_data_files)
-$(docs_data_files):
-	install -D -m 0644 $^ $@
-docs/data/intake-api/generated/events.ndjson: testdata/intake-v2/events.ndjson
-docs/data/intake-api/generated/rum_v3_events.ndjson: testdata/intake-v3/rum_events.ndjson
-docs/data/intake-api/generated/sourcemap/bundle.js.map: testdata/sourcemap/bundle.js.map
-docs/data/elasticsearch/generated/errors.json: processor/stream/test_approved_es_documents/testIntakeIntegrationErrors.approved.json
-docs/data/elasticsearch/generated/spans.json: processor/stream/test_approved_es_documents/testIntakeIntegrationSpans.approved.json
-docs/data/elasticsearch/generated/transactions.json: processor/stream/test_approved_es_documents/testIntakeIntegrationTransactions.approved.json
-docs/data/elasticsearch/generated/metricsets.json: processor/stream/test_approved_es_documents/testIntakeIntegrationMetricsets.approved.json
-docs/data/elasticsearch/generated/rum_v3_transactions.json: processor/stream/test_approved_es_documents/testIntakeRUMV3Transactions.approved.json
-docs/data/elasticsearch/generated/rum_v3_errors.json: processor/stream/test_approved_es_documents/testIntakeRUMV3Errors.approved.json
+.PHONY: copy-docs
```

Comment:
thanks/sorry - I think during the rewrite I originally had intended the "update" target to be non-PHONY, but it didn't work out due to magefile